velox/substrait/SubstraitToVeloxPlan.cpp

```diff
@@ -269,6 +366,11 @@ core::PlanNodePtr SubstraitVeloxPlanConverter::toVeloxPlan(
 core::PlanNodePtr SubstraitVeloxPlanConverter::toVeloxPlan(
     const ::substrait::ReadRel& readRel,
     std::shared_ptr<SplitInfo>& splitInfo) {
+  // emit is not allowed in TableScanNode and ValuesNode related
+  // outputs
+  if (readRel.has_common()) {
+    ValidateEmitExclusion(readRel.common());
```

Comment:
This function is used only once. It might be clear to remove it and add the check here directly.